data/styles/clear/include/Icons.mapcss

```diff
@@ -3010,50 +3010,64 @@ area|z15-[amenity=sanitary_dump_station]
 
 /* 7.1 Parking */
 
-node|z16[amenity=parking],
-area|z16[amenity=parking]
-{icon-image: parking-s.svg;icon-min-distance: 25;}
-node|z16[amenity=parking][access=permissive],
-node|z16[amenity=parking][access=private]
-{icon-image: zero-icon.svg;}
-
-node|z17[amenity=parking],
-area|z17[amenity=parking]
-{icon-image: parking-m.svg;}
-node|z17[amenity=parking][access=permissive],
-node|z17[amenity=parking][access=private]
-{icon-image: zero-icon.svg;}
-
+node|z16-[amenity=parking],
+area|z16-[amenity=parking]
+{icon-image: parking-m.svg;icon-min-distance: 5;}
 node|z18-[amenity=parking],
 area|z18-[amenity=parking]
-{icon-image: parking-m.svg;font-size: 11;}
+{font-size: 11;}
+
+node|z16-[amenity=parking][access=private],
+area|z16-[amenity=parking][access=private],
+node|z16-[amenity=parking][access=no],
+area|z16-[amenity=parking][access=no],
+node|z16-[amenity=parking][parking=underground],
+area|z16-[amenity=parking][parking=underground],
+node|z16-[amenity=parking][location=underground],
+area|z16-[amenity=parking][location=underground]
```

Comment:
'[location=underground]' line is enough;
(in fact, kothic matches only the first option from the mapping line `amenity|parking|underground;[amenity=parking][location=underground],[amenity=parking][parking=underground];;name;int_name;357;`)

Reply:
Also it makes underground parkings disappear completely?
What was your plan? To show them on later zooms than surface parkings?